dev-resources/less/modalit.less

```diff
@@ -9,13 +9,16 @@
   .modal-content {
     max-height: 90%;
     overflow-y: auto;
+    padding:32px;
```

Comment:
Tässä vois olla välilyönti kaksoispisteen jälkeen ja käyttää @valistys-32 paremman resp. hengessä. Tästä joskus koodikahveilla näin sovittiin.